salt/pillar/annotations.sls

```diff
@@ -16,6 +16,7 @@ annotations:
             client_secret: fake
         authority: null
         group: null
+    containerized: true
```

Comment:
This is the feature flag, together with adding the `annotations.containers` sls file.